src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyle = {};
+  const arrStr = sourceString.split(';');
+
+  for (const pieceArr of arrStr) {
+    // @ts-ignore
+    // eslint-disable-next-line no-console
+    console.log(pieceArr);
```

Comment:
The `console.log` statement is used for debugging purposes and should be removed in the final implementation unless explicitly required by the task.